deployments/warp_routes/EDGEN/bsc-ethereum-config.yaml

```
@@ -0,0 +1,18 @@
+tokens:
+  - chainName: bsc
+    standard: EvmHypSynthetic
+    decimals: 18
+    symbol: EDGEN
```

probably want to add the `logoURI` property for each chain within each route too
```suggestion
    symbol: EDGEN
    logoURI: /deployments/warp_routes/EDGEN/logo.svg
```